xdsl/traits.py

```diff
@@ -110,3 +112,27 @@ def verify(self, op: Operation) -> None:
                     # too; in which case it will check itself.
                     if not child_op.has_trait(IsolatedFromAbove):
                         regions += child_op.regions
+
+
+class SymbolOpInterface(OpTrait):
```

Comment:
Can you document this class directly?
And write the requirements that the operation must satisfy as well.

Reply:
- [ ] document SymbolOpInterface